test/wasmfs/wasmfs_opfs_errors_post.js

```diff
@@ -28,6 +28,10 @@ async function run_test() {
     throw "Did not get expected EIO when resizing file";
   }
 
+  if (Module._try_unlink() != 0) {
```

Comment:
We can also add another call to `Module._try_unlink()` down below after the `access.close()` to check that closing the out-of-band AccessHandle is enough to make the unlink work.

Reply:
Done. I had to add `close()` operations in the read and write calls in that test (otherwise an open fd prevents the unlink).